app/content.ts

```diff
@@ -0,0 +1,75 @@
+import * as $ from 'jquery'
```

Comment:
Why jquery?

Reply:
http://youmightnotneedjquery.com/

Reply:
Sure. Was using it to just to remove the existing script tag from DOM. Will use native API to do that